Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -22,6 +22,7 @@ import Model from "./Model.js";
 import ModelAnimationLoop from "./ModelAnimationLoop.js";
```

Comment:
This looks reasonable for now given we're under a time crunch, but it would be better to transcode the content to a Gltf3DTileContent. I opened https://github.com/CesiumGS/cesium/issues/9836 to list details like this for transcoding 1.0 formats

Reply:
I definitely second this. More and more special cases are starting to emerge making it hard to keep track of the logic.